run.sh

```diff
@@ -1,3 +1,20 @@
 #!/bin/bash
 
-docker run --rm -it -v $(pwd):/app-dir/local -w /app-dir/local contacts-fixer sh
+# Choose running machine name
+unameOut="$(uname -s)"
+case "${unameOut}" in
```

Comment:
No need for the unameOut 
```suggestion
 case "$(inane -s)" in
```